### PR TITLE
docs(github): add docs for contributing, issues, and pr's

### PR DESCRIPTION
**Pull request by brannon-darby**
# FireLoop

### What type of pull request are you creating?
- [ ] Bug Fix
- [x] Enhancement
- [x] Documentation

### How many unit test did you write for this pull request?
0

#### Write a reason if none (e.g just fixed a typo):
documentation

### Please add a description for your pull request:
GitHub docs to standardize commits, issues, pr's, and contributions

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Adding Issues
+If you find any bug or enhancement, you are always welcome to create a [New Issue](https://github.com/mean-expert-official/fireloop-starter/issues).
+
+## Pull Request
+If you would like to contribute by adding new features, enhancements or by fixing bugs; please consider that you need to create an issue prior your pull request. This is important for tracking purposes in CHANGELOG.
+
+**IMPORTANT: Make sure you create a branch from `development` and then create the pull request also to development and not to master.**
+
+Please send a [GitHub Pull Request](https://github.com/mean-expert-official/fireloop-starter/pull/new/development) with a clear list of what you've done (read more about [pull requests](http://help.github.com/pull-requests/)). When you send a pull request, we will love you forever if you include RSpec examples. We can always use more test coverage. Please make sure all of your commits are atomic (one feature per commit).
```
Comment (jonathan-casarrubias) [inline .github/CONTRIBUTING.md:24]: I believe in this repo the branch is named develop and not development.

Comment (beeman) [reply .github/CONTRIBUTING.md:24]: @jonathan-casarrubias you're right. But let's keep it the same like other projects in this org. If they use `development`, so should this repo :)

Comment (jonathan-casarrubias) [reply .github/CONTRIBUTING.md:24]: cool, that is my preference though... so lets make it development afterwards

Comment (brannon-darby) [reply .github/CONTRIBUTING.md:24]: @jonathan-casarrubias - are you saying change it to develop for now, then switch it to development later?

Comment (jonathan-casarrubias) [reply .github/CONTRIBUTING.md:24]: Try to migrate from develop to development as soon as it is possible, not a priority though... but at the end it should be "development" for  consistency

Comment (beeman) [reply .github/CONTRIBUTING.md:24]: I will do so when I merged through all the PR's. Otherwise those have to be recreated against the `development` branch

**Pull request merged**